test_scripts/Defects/5_0/885_For_some_RPCs_Core_does_not_unsubscribe_from_softbuttons_after_receiving_a_response/AlertManeuver.lua

```diff
@@ -0,0 +1,111 @@
+    "NewTestCaseGroup"}
+end
+
+local function AlertManeuver()
+    local params = {
+        ttsChunks = {
+          { type = "TEXT", text = "pathToFile" }
+        },
+        softButtons = {
+            { type = "TEXT", softButtonID = 4, text = "text" }
+        }
+    }
+    local responseDelay = 3000
+    local cid = common.getMobileSession():SendRPC("AlertManeuver", params )
+    EXPECT_HMICALL("Navigation.AlertManeuver",{
+        softButtons = {
+            {type = "TEXT",
+            softButtonID = 4}
+        }
+    })
+    :Do(function(_, data)
+        local function alertManeuverResponse()
+            common.getHMIConnection():SendResponse(data.id, "Navigation.AlertManeuver", "SUCCESS", { })
+        end
+        RUN_AFTER(alertManeuverResponse, responseDelay)
+    end)
+    params.appID = common.getHMIAppId()
+    common.getHMIConnection():ExpectRequest("TTS.Speak", {
+        ttsChunks = {
+          { type = "TEXT", text = "pathToFile" }
+        }
+    })
+	:Do(function(_,data)
+		common.getHMIConnection():SendNotification("TTS.Started")
+		local function SpeakResponse()
+			common.getHMIConnection():SendResponse(data.id, "TTS.Speak", "SUCCESS", { })
+			common.getHMIConnection():SendNotification("TTS.Stopped")
+		end
+		local function ButtonEventPress()
+            common.getHMIConnection():SendNotification("Buttons.OnButtonEvent",
+			{ name = "CUSTOM_BUTTON", mode = "BUTTONDOWN", customButtonID = 4, appID = common.getHMIAppId()
+			})
+            common.getHMIConnection():SendNotification("Buttons.OnButtonEvent",
+			{ name = "CUSTOM_BUTTON", mode = "BUTTONUP", customButtonID = 4, appID = common.getHMIAppId()
+			})
+            common.getHMIConnection():SendNotification("Buttons.OnButtonPress",
+			{ name = "CUSTOM_BUTTON", mode = "SHORT", customButtonID = 4, appID = common.getHMIAppId()
+			})
+		end
+
+		RUN_AFTER(ButtonEventPress, 1000)
+        RUN_AFTER(SpeakResponse, 2000)
+	end)
+    common.getMobileSession():ExpectNotification("OnButtonEvent",
+    {buttonName = "CUSTOM_BUTTON", buttonEventMode = "BUTTONDOWN", customButtonID = 4 },
+    {buttonName = "CUSTOM_BUTTON", buttonEventMode = "BUTTONUP", customButtonID = 4 })
+    :Times(2)
+    common.getMobileSession():ExpectNotification("OnButtonPress",
+    {buttonName = "CUSTOM_BUTTON", buttonPressMode = "SHORT", customButtonID = 4 })
+
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("Update ptu", common.policyTableUpdate, { pTUpdateFunc })
+runner.Step("Activate App", common.activateApp)
+
+-- [[ Test ]]
+runner.Title("Test")
+runner.Step("AlertManeuver with soft buttons", AlertManeuver)
```

Comment:
@HSavynetska Current test sequence does not check that SDL stops processing softButton events once a response is received for the RPC.
Please correct test sequence.